### PR TITLE
Migrate to GH Actions

### LINKED ISSUE #17: global: migration to github-actions from travis

### PR DESCRIPTION
**Pull request by avivace**
This PR was prepared by an [automated action](https://github.com/inveniosoftware/automation-tools/tree/master/automation_tools/scripts/ga-migration). Closes #17. Fails because of some combinations timing out (#19)

### CODE REVIEW

File: run-tests.sh [hunk 1]
```diff
@@ -1,13 +1,29 @@
...
+trap cleanup EXIT
+
+
+python -m check_manifest --ignore ".*-requirements.txt"
+python -m sphinx.cmd.build -qnNW docs docs/_build/html
+eval "$(docker-services-cli up --cache ${CACHE:-redis} --mq ${MQ:-rabbitmq} --env)"
 python -m pytest
+tests_exit_code=$?
 python -m sphinx.cmd.build -qnNW -b doctest docs docs/_build/doctest
+exit "$tests_exit_code"
```
Comment (ParthS007) [inline run-tests.sh:29]: Minor: Missing empty line

File: setup.py [hunk 2]
```diff
@@ -16,7 +16,7 @@
 history = open('CHANGES.rst').read()
 
 tests_require = [
-    'pytest-invenio>=1.4.0',
+    "pytest-invenio>=1.4.0"
```
Comment (ParthS007) [inline setup.py:19]: Minor: 
```suggestion
    'pytest-invenio>=1.4.0'
```

**Review by ParthS007 — COMMENTED** [review]
Thanks for the PR. LGTM, Tests Passing. I canceled the workflow because it was taking very long. Left some minor comments.

File: .github/workflows/pypi-publish.yml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install setuptools wheel
+
+      - name: Build package
+        # Remove `compile_catalog` if the package has no translations.
+        run: |
+          python setup.py compile_catalog sdist bdist_wheel
```
Comment (diegodelemos) [inline .github/workflows/pypi-publish.yml:28]: The module doesn't have translations:
```suggestion
          python setup.py sdist bdist_wheel
```

Perhaps you can rerun the ga-migration script as this is fixed there.

Comment (avivace) [reply .github/workflows/pypi-publish.yml]: @diegodelemos Done

**Review by ParthS007 — APPROVED** [review]

File: .github/workflows/tests.yml [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+    inputs:
+      reason:
+        description: 'Reason'
+        required: false
+        default: 'Manual trigger'
+
+jobs:
+  Tests:
+    runs-on: ubuntu-20.04
+    strategy:
```
Comment (ParthS007) [inline .github/workflows/tests.yml:29]: Add `timeout`

**Review by ParthS007 — CHANGES_REQUESTED** [review]

**Review by ParthS007 — APPROVED** [review]

**Comment by diegodelemos** [comment]
CI failures to be followed up at https://github.com/inveniosoftware/invenio-queues/issues/19.

**Pull request merged**